module/VuFindApi/src/VuFindApi/Controller/ApiControllerFactory.php

```diff
@@ -44,6 +44,18 @@
  */
 class ApiControllerFactory implements FactoryInterface
 {
+    /**
+     * API controllers to register
+     *
+     * @var array
+     */
+    protected $apiControllers = [
```

Comment:
Is this property unused?

Reply:
Yes, it was left over as a part of previous iteration.